src/calculateRentalCost.js

```diff
@@ -29,6 +29,28 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const costDay = 40;
+  const weekDiscount = 50;
+  const threeDayDiscount = 20;
```

Comment:
```suggestion
  const bigDiscount = 50;
  const smallDiscount = 20;
```
It is recommended not to connect the variable's name with values that can change later. For example. we might need to change the requirement for small discount from three days to two days later